composer.json

```diff
@@ -20,5 +20,8 @@
             "@statistics"
         ],
         "wikidata": "php scripts/wikidata.php"
+    },
+    "config": {
+        "process-timeout": 800
```

Comment:
I would prefer not to specify any global process timeout. I guess 800 would be enough for most city but also can be too little for some.
I usually run `composer run update-data --timeout=0 -- --city=brussels`.
I'll disable the timeout for the `update-data` script only.

Reply:
Done 😄  
https://github.com/openknowledgebe/equalstreetnames/blob/a2ea526d2983a5fb1e9cf3844ed17b6d0a110856/composer.json#L16